lib/channel-websocket/tsconfig.json

```diff
@@ -3,6 +3,9 @@
   "compilerOptions": {
     "rootDir": "./src"
   },
+  "include": [
+    "src/**/*"
+  ],
```

Comment:
When working on the stdout and stderr coming from `compile-tsc`, this came up.

I ran into a problem where TSC tried to compile files in `dist`. It refused to do it, because files already existed.
This will ensure `tsc` will only compiled things in `src`.